outlines/models/llamacpp.py

```diff
@@ -288,6 +289,16 @@ def llamacpp(
     if "verbose" not in llamacpp_model_params:
         llamacpp_model_params["verbose"] = False
 
+    # TODO: Remove when https://github.com/ggerganov/llama.cpp/pull/5613 is resolved
+    if "tokenizer" not in llamacpp_model_params:
```

Comment:
It would be good to add a test that will fail when the issue is solved on `llama.cpp`’s end.